### PR TITLE
Release 2.7.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by alexjfisher** [comment]
IMO new changelog is a mess compared to what it is replacing.

**Comment by bastelfreak** [comment]
uhm? Both got generated with gcg. I don't see a big difference.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,34 +1,1209 @@
-# Change Log
```
Comment (alexjfisher) [inline CHANGELOG.md]: The old changelog was using a history.md for old releases and gcg only for the last few.  I think I can fix this.

File: HISTORY.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## 2017-04-24 - Release 2.4.1 ([diff](https://github.com/locp/cassandra/compare/2.4.0...2.4.1))
+## [2.4.1](https://github.com/voxpupuli/puppet-cassandra/compare/2.4.0...2.4.1) (2017-04-24)
```
Comment (alexjfisher) [inline HISTORY.md]: The *exact* format of the first release in the HISTORY.md has to be correct for gcg to work correctly.  (even a trailing space breaks it!)

**Pull request merged**